_includes/common-and-popular.html

```diff
@@ -24,7 +24,7 @@ <h3 class="va-h-ruled">Popular on vets.gov</h3>
         <li>
           <a href="/gi-bill-comparison-tool/">View education benefits available by school</a>
         </li>
-        <li><a target="_blank" href="https://www.veteranscrisisline.net/" class="external">Contact the Veterans Crisis Line</a></li>
+        <li><a target="_blank" href="https://www.veteranscrisisline.net/" class="external" onclick="dataLayer.push({'event': 'popular-link-clicked'});">Contact the Veterans Crisis Line</a></li>
```

Comment:
Instead of adding an onclick attribute, can we have some sort of data attribute that we look for and forward to Google Analytics? Like `data-va-event="popular-link-clicked"` and then a jQuery (or equivalent) handler for `$("[data-va-event]").on(...)`? That way, if a user clicks the link but for some reason JavaScript hasn't loaded (slow network, e.g.) there won't be an error in the console.